extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -463,6 +466,13 @@ export class AmpA4A extends AMP.BaseElement {
           if (!fetchResponse || !fetchResponse.arrayBuffer) {
             return null;
           }
+          // If the response has response code 204, collapse it.
+          if (fetchResponse.status == 200) {
+            // Necessary, as uiHandler will ignore the request to display no
+            // content otherwise.
+            this.uiHandler.forceNoContentUI();
```

Comment:
Sheer paranoia: by the way that `this.uiHandler` is initialized, it _should_ never be null here.  But it's not type-enforced and, because it's public, we can't strictly guarantee that it hasn't been altered.  So for paranoia, could you toss a `dev().assert(this.uiHandler)` in front?